patches/chrome-tools-build-win-create_installer_archive.py.patch

```diff
@@ -39,7 +39,7 @@ index 732479bbf5975289bd251ab6d41c8dda7e808a50..b85b6ffd7bab57a5da1d6819bccbaab0
             'with the installer archive {x86|x64}.')
    parser.add_option('-v', '--verbose', action='store_true', dest='verbose',
                      default=False)
-+  parser.add_option('--skip_signing', default=False)
++  parser.add_option('--skip_signing', action='store_true', default=False)
```

Comment:
The lack of `action='store_true'` results in missing hidpi resources with a local build (with `--skip_signing`):
![image (6)](https://user-images.githubusercontent.com/45488748/142976880-c5980de2-a35c-4900-96a2-b214eab380e9.png)